Resources/Maps/_DV/Shuttles/USSPS_Orit.yml

```diff
@@ -0,0 +1,2301 @@
+meta:
+  format: 7
+  category: Grid
+  engineVersion: 255.1.0
+  forkId: ""
+  forkVersion: ""
+  time: 05/19/2025 19:27:19
+  entityCount: 335
+maps: []
+grids:
+- 1
+orphans:
+- 1
+nullspace: []
+tilemap:
+  2: Space
+  3: FloorSteel
+  0: FloorTechMaint
+  5: FloorWhite
+  4: Lattice
+  1: Plating
+entities:
+- proto: ""
+  entities:
+  - uid: 1
+    components:
+    - type: MetaData
+      name: USSPS Orit
+    - type: Transform
+      pos: -2.015625,-1.5473902
+      parent: invalid
+    - type: BecomesStation
+      id: MapRenderer
```

Comment:
why did you add BecomesStation for a ship

also remove offset/invalid from all these grid transforms

Reply:
BecomesStation for map rendering because map rendering only works on station maps and not shuttles

Reply:
that limitation wont exist after upstream merge just remove it

Reply:
ok